meta-resin-common/classes/image-resin.bbclass

```diff
@@ -237,7 +237,7 @@ QUIRK_FILES ?= " \
     "
 resin_root_quirks () {
     # Quirks
-    # We need to save some files that docker shadows with bind mounts
+    # We need to save some files that container engine shadows with bind mounts
```

Comment:
Maybe this should be "that THE container engine" instead?

Reply:
Fixed